### ISSUE TITLE
npm install fails on Windows cmd

### ISSUE THREAD

**Issue by sxmon** [Teradata/covalent-quickstart#68]
#### Bug Report

On Windows, clone the repo and npm i. 
Error =     '.' is not recognized as an internal or external command,

#### What is the expected behavior?
Installation works.

#### What is the motivation / use case for changing the behavior?
Availability to windows users.

##### Other information
Fix = "webdriver-manager update && bash ./scripts/protractor.sh" in package.json

**Comment by jotatoledo** [comment]
Cant reproduce your issue. 
I cloned the repo and executed `npm i` with no problems.
Did u ran `npm i` or `npm i. `? No error log was generated?

**Comment by jotatoledo** [comment]
Yeah the execution problem is related to the `postinstall` script in package.json

**Comment by cnrudd** [comment]
I'm on Windows 10.  
Changing postinstall to:
`"postinstall": "npm run webdriver-update && bash ./scripts/protractor.sh"`

fixed the issue for me

**Comment by wakjoko** [comment]
my fix in package.json was:

`"postinstall": "webdriver-manager update && bash ./scripts/protractor.sh",`

**Comment by mikehutter** [comment]
I am also on Windows 10 and I tried using the solution proposed by @cnrudd and I got the following:

./scripts/protractor.sh: line 2: $'\r': command not found
: No such file or directoryne 3: cd: ./node_modules/protractor/node_modules
./scripts/protractor.sh: line 4: $'\r': command not found
./scripts/protractor.sh: line 10: syntax error: unexpected end of file

and found the solution to that here:

https://learningintheopen.org/2013/03/07/microsoft-windows-cygwin-error-r-command-not-found/
